src/swarm-tui/Swarm/TUI/Model/Repl.hs

```diff
@@ -379,9 +381,18 @@ replPromptEditor :: Lens' REPLState (Editor Text Name)
 replPromptText :: Lens' REPLState Text
 replPromptText = lens g s
  where
-  g r = r ^. replPromptEditor . to getEditContents . to T.concat
+  g r = r ^. replPromptEditor . to getEditContents . to T.unlines
```

Comment:
Is the unlines unused currently since the REPL is always one line?

Reply:
Yes, that's true.  But since the zipper returns a list of lines, `T.unlines` is really the correct thing to do (I noticed that it was `T.concat`; I think previously I misunderstood what the return type of `getEditContents` represented).